docs/source/configuration/persisted-queries.mdx

```diff
@@ -0,0 +1,183 @@
+
+## Implementation
+
+Enabling operation safelisting has a few steps:
+
+  1. PQL creation and linking
+  2. Router configuration
+  3. Preregister trusted operations
+  4. Client updates
+
+This article details the router configuration step.
+For more information on other configuration aspects, see the [GraphOS persisted queries documentation](/graphos/operations/persisted-queries).
+
+## Router configuration
+
+After client teams publish operations to a graph's PQL, you can configure router instances to fetch and use the PQL by following these steps:
+
+1. Ensure your router instances are ready to work with PQLs:
+    - Make sure you're using version `1.25.0` or later of the Apollo Router.
+    - Make sure your router instances are [connected to your GraphOS Enterprise organization](../enterprise-features/#enabling-enterprise-features) and that they're associated with a variant that your PQL is linked to.
+
+2. Set your desired security level in your router's YAML config file. For supported options, see [router security levels](#router-security-levels).
+
+3. Deploy your updated router instances to begin using your PQL.
+
+### Router security levels
+
+The Apollo Router supports the following [security levels](#security-levels), in increasing order of restrictiveness:
+
+- **Performance-only**: Clients can optionally execute an operation on your router by providing the operation's PQL-specified ID.
+    - All other levels also provide this core capability.
+    - This level provides no operation safelisting.
+- **Audit mode**: Executing operations by providing a PQL-specified ID is still optional, but the router also logs any unregistered operations.
+    - The level serves as a dry run and helps you identify operations you may still need to register before turning on safelisting.
+- **Safelisting with operation strings allowed**: The router _rejects_ any incoming operations that aren't present in its PQL.
+    - Before moving to this security level, ensure _all_ your client operations are present in your PQL.
+- **Safelisting with IDs only**: The router rejects any incoming operations that aren't present in its PQL, _and_ clients _must_ execute an operation by providing its PQL-specified ID.
+    - Before moving to this security level, ensure _all_ your clients execute operations by providing their PQL-specified ID.
+
+When you first adopt persisted queries, you should start with a less restrictive security such as [audit mode](#audit-mode-dry-run). You can then enable increasingly restrictive levels after your teams has updated all clients.
+
+See below for sample YAML configurations for each level.
+
+#### Performance-only
+
+To use persisted queries _only_ to reduce network bandwidth and latency (_not_ for safelisting), add the following minimal configuration:
+
+```yaml title="router.yaml"
+preview_persisted_queries:
+  enabled: true
+apq:
+  enabled: true # APQ can be enabled
```

Comment:
I think these should be removed (as we did in https://github.com/apollographql/docs/pull/496)